v1/pipeline.json

```diff
@@ -50682,6 +50691,265 @@
             "type" : "string"
           }
         }
+      },
+      "Notification Rules" : {
```

Comment:
Why the space between Notification Rules? Lets make it `NotificationRules`. Same for other places as well

Reply:
done